### PR TITLE
1.8.2 release

### PR DESCRIPTION
**Pull request by edenhill**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,26 @@
...
 
+## v1.8.0
+
+v1.8.0 is a maintenance release with the following fixes and enhancements:
+
+ - **Python 2.7 binary wheels are no longer provided.**
+   Users still on Python 2.7 will need to build confluent-kafka from source
+   and install librdkafka separately, see [README.md](README.md#Prerequisites)
+   for build instructions.
+ - Added `use.latest.version` and `skip.known.types` (Protobuf) to
```
Comment (mhowlett) [inline CHANGELOG.md:11]: i don't think the protobuf framing incompatibility got fixed?

**Review by mhowlett — COMMENTED** [review]
lgtm if the protobuf framing incompatibility is fixed.

**Comment by mhowlett** [comment]
it would be less confusing if the version was 1.8.2, to match librdkafka.

File: CHANGELOG.md (hunk 1 above)
Comment (edenhill) [reply CHANGELOG.md:11]: Is there a PR for it that Ive missed?

**Comment by edenhill** [comment]
Bumped to 1.8.2

**Comment by jonchiu** [comment]
> lgtm if the protobuf framing incompatibility is fixed.

Did we get this fixed?

**Comment by mhowlett** [comment]
> > lgtm if the protobuf framing incompatibility is fixed.
> 
> Did we get this fixed?

yes https://github.com/confluentinc/confluent-kafka-python/pull/1256

**Pull request merged**